doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -145,6 +145,11 @@ Bluetooth Mesh
  * Added metadata as optional parameter for models Light Lightness Server, Light HSL Server, Light CTL Temperature Server, Sensor Server, and Time Server.
    To use the metadata, enable the :kconfig:option:`CONFIG_BT_MESH_LARGE_COMP_DATA_SRV` Kconfig option.
 
+* Removed:
+
+  * Removed the ``BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE`` Kconfig option as the old APIs based on the :c:struct:`sensor_value` type are removed.
```

Comment:
Btw, is :c:struct:`sensor_value` also removed? Because we cannot link to it using :c:struct: anyways if it is removed. Instead, we can wrap it with ``.

Reply:
it is still there, we only removed the APIs using it.